resources/schemas/dbscripts/sqlserver/snd-21.000-21.001.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE exp.PropertyValidator SET TypeURI = 'urn:lsid:labkey.com:PropertyValidator:textlength'
+WHERE PropertyId IN (
+    SELECT pd.PropertyId
+    FROM exp.PropertyValidator pv
+    JOIN exp.PropertyDescriptor pd on pd.PropertyId = pv.PropertyId
+    where pv.TypeURI = 'urn:lsid:labkey.com:PropertyValidator:length' and pd.PropertyURI like '%package-snd%'
```

Comment:
The TypeURI filter should move outside of the inner select. A property might have both length and required validators, for example.

Also, can we we make the PropertyURI like clause a little longer? Will there be a colon or dash immediately before or after package-snd, for example? It's probably low risk of overlap, but the current filter will match anywhere at all in the URI, including a property with that name.

Reply:
Updated